pkg/kyamls/helpers_test.go

```diff
@@ -0,0 +1,30 @@
+package kyamls
```

Comment:
Thanks for adding tests, can we add a test case also for the case where it errors out (we can use table driven) for both labels and annotations?

Reply:
Yup.